docs/deploy_logging.md

```diff
@@ -0,0 +1,49 @@
+# Deploy openshift-logging on cluster
+
+### Pre-requisites before starting to deploy openshift-logging
+
+###### It is recommended to have minimum m5.xlarge setup.
+
+The images for the openshift-logging need to be pulled from the quay,
+sync the images with the internal registry and update the cluster with the new images for containers.
+
+
+Note: Steps are to be followed till OCP 4.2 is GAed, These steps can be skipped after that
+
+1. Create a quay.io account [https://quay.io]
+
+2. Add your quay username in the sheet given below and wait for the invite sent
+from quay.io ocp-repo maintainers
+https://docs.google.com/spreadsheets/d/1OyUtbu9aiAi3rfkappz5gcq5FjUbMQtJG4jZCNqVT20/edit#gid=0
+
+3. Generate the quay_token with the command
+
+```
+curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '
```

Comment:
The user needs to replace $USER and $PASSWD by login information from quay or needs to be set to variables before executing the command. This should be also documented to be precise for user.

Maybe you can directly write to > ocs-ci/data/quay.token  (You have to mention that command needs to be executed from root of ocs-ci repository)